package.json

```diff
@@ -40,5 +40,8 @@
     "git-commit-template": "^1.0.5",
     "husky": "^4.2.5",
     "lerna": "^3.20.2"
+  },
```

Comment:
why you can this new package here

Reply:
It's used for logging. It writes telemetry logs to a file instead of the buffer.